spec/lib/miq_automation_engine/models/miq_ae_password_spec.rb

```diff
@@ -40,4 +40,10 @@
       end
     end
   end
+
+  describe ".key_root" do
+    it "has key_root set" do
+      expect(MiqAePassword.key_root).to be
```

Comment:
This is a string, and `to be` is equivalent to `not_to be_nil` or `to be_present`
think this is what we want. It was the simplest green/red/green I could get.